### PR TITLE
[WIP] Feature/jshint update

### PR DESCRIPTION
**Pull request by mschultheiss83**
- i added .gitattributes to force on windows for LF endings, - i had problems with eslint & jshint
- i added to .jshintrc the "unused" rule
- handleDiedCreeps reworked for lower complexity + similar code roles.[role].died

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* text eol=lf
+*.jpg binary
+*.png binary
```
Comment (TooAngel) [inline .gitattributes]: nitpicking: newline at the end of the file

File: src/prototype_creep_startup_tasks.js [hunk 2]
```diff
@@ -198,8 +201,8 @@ Creep.prototype.repairStructure = function() {
       for (const spawn of spawns) {
         let found = false;
         let rampart;
-        const structures = spawn.pos.lookFor(LOOK_STRUCTURES);
-        for (const structure of structures) {
+        structures = spawn.pos.lookFor(LOOK_STRUCTURES);
```
Comment (TooAngel) [inline src/prototype_creep_startup_tasks.js]: I think `structures` can stay `const`

File: src/role_squadheal.js [hunk 3]
```diff
@@ -107,4 +107,4 @@ roles.squadheal.execute = function(creep) {
   creep.log('Execute!!!');
   creep.heal(creep);
   creep.moveRandom();
-};
+};
```
Comment (TooAngel) [inline src/role_squadheal.js]: nitpicking: newline

File: src/prototype_creep_startup_tasks.js (hunk 2 above)
Comment (mschultheiss83) [reply src/prototype_creep_startup_tasks.js]: my ide says no eslint says no

**Comment by mschultheiss83** [comment]
@TooAngel please review again

**Comment by mschultheiss83** [comment]
@Nevrdid @samogot please review

File: src/prototype_creep_startup_tasks.js (hunk 2 above)
Comment (Nevrdid) [reply src/prototype_creep_startup_tasks.js]: better keep it out of for but with a let

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 326/502
    Coefficient: 0.649402390438
    Merging in 18 days 19 hours
    Age 0 days 11 hours

**Comment by TooAngel** [comment]
Suggesting to rebase to fewer commits

**Comment by democratic-collaboration** [comment]
A new review, yeah.

            Votes: 93/529
            Coefficient: 0.175803402647
            Merging in 65 days 10 hours
            Age 0 days 12 hours

**Comment by mschultheiss83** [comment]
closing this one and reopen in new tiny pull requests

**Pull request closed without merging**